Documentation/Testing/Tutorial/Acceptance.rst

```diff
@@ -0,0 +1,267 @@
+:file:`.ddev/docker-compose.chrome.yaml.inactive` has to be renamed to :file:`.ddev/docker-compose.chrome.yaml`.
+By default acceptance tests are disabled because they slow down other tests significantly.
+
+Next, after adding codeception as require-dev dependency in :file:`composer.json`, we need a
+basic :file:`Tests/codeception.yml` file:
+
+.. code-block:: yaml
+
+    namespace: Bk2k\SiteIntroduction\Tests\Acceptance\Support
+    suites:
+      acceptance:
+        actor: AcceptanceTester
+        path: .
+        modules:
+          enabled:
+            - Asserts
+            - WebDriver:
+                url: https://introduction.ddev.site
+                browser: chrome
+                host: ddev-introduction-chrome
+                wait: 1
+                window_size: 1280x1024
+    extensions:
+      enabled:
+        - Codeception\Extension\RunFailed
+        - Codeception\Extension\Recorder
+
+    paths:
+      tests: Acceptance
+      output: ../var/log/_output
+      data: .
+      support: Acceptance/Support
+
+    settings:
+      shuffle: false
+      lint: true
+      colors: true
+
+This tells codeception there is a selenium instance at `ddev-introduction-chrome` with chrome,
+the website is reachable as :samp:`https://introduction.ddev.site`, it enables some codeception plugins
+and specifies a couple of logging details. The `codeception documentation <https://codeception.com/>`_
+goes into details about these.
+
+Now we need a simple first test which is added as :file:`Tests/Acceptance/Frontend/FrontendPagesCest.php`:
+
+.. code-block:: php
+   :caption: EXT:site_introduction/Tests/Acceptance/Frontend/FrontendPagesCest.php
+
+    <?php
+    declare(strict_types = 1);
+    namespace Bk2k\SiteIntroduction\Tests\Acceptance\Frontend;
+    use Bk2k\SiteIntroduction\Tests\Acceptance\Support\AcceptanceTester;
+    class FrontendPagesCest
+    {
+        /**
+         * @param AcceptanceTester $I
+         */
+        public function firstPageIsRendered(AcceptanceTester $I)
+        {
+            $I->amOnPage('/');
+            $I->see('Open source, enterprise CMS delivering  content-rich digital experiences on any channel,  any device, in any language');
+            $I->click('Customize');
+            $I->see('Incredible flexible');
+        }
+    }
+
+It just calls the homepage of our instance, clicks one of the links and verifies some text is
+shown. Straight, but enough to see if the basic instance does work.
+
+Ah, and we need a "Tester" in the `Support directory <https://github.com/TYPO3-Documentation/site-introduction/tree/main/Tests/Acceptance/Support>`_.
+
+That's it. We can now execute the acceptance test suite by executing a command in the
+ddev PHP container:
+
+.. code-block:: shell
+
+    lolli@apoc /var/www/local/site-introduction $ ddev exec bin/codecept run acceptance -d -c Tests/codeception.yml
+    Codeception PHP Testing Framework v2.5.6
```

Comment:
Versions seems quote outdated - at least for current project.s phpunit7/codeception 2.x seems like TYPO3 v9 or earlier ....

Reply:
I just moved this content unchanged, we will deal with it in a follow up